Cargo.toml

```diff
@@ -79,7 +80,24 @@ winapi = { version = "0.3.9", features = [
     "timeapi",
     "mmsystem",
 ] }
-native-windows-gui = { version = "1.0.12", default_features = false }
+windows-sys = { version = "0.52.0", features = [
+    "Win32_Devices_DeviceAndDriverInstallation",
+    "Win32_Devices_Usb",
+    "Win32_Foundation",
+    "Win32_Graphics_Gdi",
+    "Win32_Security",
+    "Win32_System_Diagnostics_Debug",
+    "Win32_System_Registry",
+    "Win32_System_Threading",
+    "Win32_UI_Controls",
+    "Win32_UI_Shell",
+    "Win32_UI_WindowsAndMessaging",
+], optional=true }
+win_dbg_logger = { path = "win_dbg_logger", optional = true }
+native-windows-gui = { version = "1.0.13", default_features = false}
+native-windows-derive = { version = "1.0.5", default_features = false, optional = true }
+lazy_static = { version = "1.4.0", optional = true }
```

Comment:
Please avoid `lazy_static` if possible and use the std `OnceLock` instead. Not that `lazy_static` is a bad crate; I want to minimize dependencies.

Reply:
ok